extensions/README.md

```diff
@@ -0,0 +1,11 @@
+This folder contains the scripts for various extensions & plugins used in Oppia codebase.
```

Comment:
They aren't really "scripts", right?

Reply:
Updated.

Reply:
It still refers to "scripts", but these aren't scripts.

Reply:
My bad, fixed now.